### PR TITLE
Configure MiMa

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,7 +15,7 @@ val Scala213 = "2.13.8"
 ThisBuild / crossScalaVersions := Seq(Scala213, "3.1.1")
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
-lazy val root = project.in(file(".")).aggregate(servlet, examples)
+lazy val root = project.in(file(".")).aggregate(servlet, examples).enablePlugins(NoPublishPlugin)
```
Comment (rossabaker) [inline build.sbt:18]: This seems like it's missing from the g8 template?

**Review by rossabaker — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (armanbilge) [reply build.sbt:18]: The g8 uses `tlCrossRootProject` which does all this automatically.
https://github.com/typelevel/typelevel.g8/blob/6c846d47f9a8fc601a06cf5474f496af90573fe8/src/main/g8/build.sbt#L22

It's technically designed for cross-platform builds, but it's harmless and does a lot of good things by default anyway.

Comment (rossabaker) [reply build.sbt:18]: Should this module then be using that but just with JVMPlatform?

Comment (armanbilge) [reply build.sbt:18]: You don't even need to specify `JVMPlatform`. You will have to re-generate the workflow.
```suggestion
lazy val root = tlCrossRootProject.aggregate(servlet, examples)
```

Comment (rossabaker) [reply build.sbt:18]: Oh, I see.  I thought that would cross all the modules, not just the aggregate.

Comment (armanbilge) [reply build.sbt:18]: A `tlCrossRootProject` enables the usual defaults for root projects (no-publish etc.) and configures the CI matrix for cross-building, depending on what sorts of projects get aggregated into it.

**Comment by armanbilge** [comment]
CI-skipping that will get you into trouble, the workflow needs to be regenerated!

**Comment by rossabaker** [comment]
Yep.  It's two commits because the suggestion was in the browser and the regenerate was in my shell and I didn't want two builds.

**Pull request merged**